presto-main/src/test/java/io/prestosql/metadata/TestInformationSchemaMetadata.java

```diff
@@ -217,6 +218,36 @@ public void testInformationSchemaPredicatePushdownOnCatalogWiseTables()
         assertFalse(result.isPresent());
     }
 
+    @Test
+    public void testInformationSchemaPredicatePushdownForEmptyNames()
```

Comment:
Can you please also add a test to `TestInformationSchemaConnector` where you run actual query and verify the result?